test/source/tests/compose.ts

```diff
@@ -1342,6 +1342,34 @@ export const defineComposeTests = (testVariant: TestVariant, testWithBrowser: Te
       await expectRecipientElements(composePage, { to: [], cc: [], bcc: [] });
     }));
 
+    ava.default('compose - reply - from === acctEmail', testWithBrowser('compatibility', async (t, browser) => {
+      const appendUrl = 'threadId=17d02296bccd4c5c&skipClickPrompt=___cu_false___&ignoreDraft=___cu_false___&replyMsgId=17d02296bccd4c5c';
+      const composePage = await ComposePageRecipe.openStandalone(t, browser, 'compatibility', { appendUrl, hasReplyPrompt: true });
+      await composePage.waitAndClick('@encrypted-reply', { delay: 1 });
+      await expectRecipientElements(composePage, { to: ['flowcrypt.compatibility@gmail.com', 'vladimir@flowcrypt.com'], cc: [], bcc: [] });
```

Comment:
Strange that `flowcrypt.compatibility@gmail.com` is included in `to` with "reply", whereas it's not included with "reply all". Is there a purpose of such behaviour?

Reply:
My goal here was to align the behavior with Gmail so there will be no surprises for our users.

In this particular case `flowcrypt.compatibility@gmail.com` sends the message 

- `to` 
  - `flowcrypt.compatibility@gmail.com` (itself, `from === acctEmail`)
  - `vladimir@flowcrypt.com`
- `cc`
  - `limon.monte@gmail.com`
- `bcc`
  - `sweetalert2@gmail.com`

Here's the Gmail's behavior which is aligned with the test's expectations:

<img width="752" alt="CleanShot 2021-11-11 at 10 01 55@2x" src="https://user-images.githubusercontent.com/6059356/141260624-937ebc5c-3349-4ad1-8821-cf52f9344b6a.png">

You can test it by yourself by logging into `flowcrypt.compatibility@gmail.com` and opening this thread https://mail.google.com/mail/u/4/#inbox/WhctKKXHCWWrrFvSmnhdCjDHBtHBbzrqmFPQWKBVqftJdVQFxGtTTPDWZrvZKPXlJpckmNL